.github/workflows/e2eLocal.yml

```diff
@@ -0,0 +1,87 @@
+
+name: Run end-to-end tests
+
+on:
+  push:
+    branches:
+      - "**"
+jobs:
+
+  cypress-local-env:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Set Swap Space
+      uses: ./.github/actions/set-swap-space
+      with:
+        swap-size-gb: 10
+
+    - name: Generate certs
+      run: |
+        sudo apt install libnss3-tools
+        brew install mkcert
```

Comment:
Whoa, does this ubuntu image have homebrew installed by default?

Reply:
Yes! I was surprised too.